### PR TITLE
Update README.md with information about GraphQL API EAP

### PR DESCRIPTION
**Pull request by LackOfMorals**
# Description

> **Note**
> 
> Please provide a description of the work completed in this PR below

## Complexity

> **Note**
>
> Please provide an estimated complexity of this PR of either Low, Medium or High

Complexity:

# Issue

> **Note**
>
>  Please link to the GitHub issue(s) in which the proposal for this work was discussed
>  
>  To link to multiple issues, use full syntax for each, for example `Closes #1, closes #2, closes #3`

Closes 

# Checklist

The following requirements should have been met (depending on the changes in the branch):

- [ ] Documentation has been updated
- [ ] TCK tests have been updated
- [ ] Integration tests have been updated
- [ ] Example applications have been updated
- [ ] New files have copyright header
- [ ] CLA (https://neo4j.com/developer/cla/) has been signed

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,14 +2,33 @@
...
+
 ![Neo4j + GraphQL](./images/banner.png)
 
 <p align="center">
   <a href="https://discord.gg/neo4j">
     <img alt="Discord" src="https://img.shields.io/discord/787399249741479977?logo=discord&logoColor=white">
   </a>
   <a href="https://community.neo4j.com/c/drivers-stacks/graphql/33">
-    <img alt="Discourse users" src="https://img.shields.io/discourse/users?logo=discourse&server=https%3A%2F%2Fcommunity.neo4j.com">
+    <img alt="Discourse users" src="https://img.shields.iåo/discourse/users?logo=discourse&server=https%3A%2F%2Fcommunity.neo4j.com">
```
Comment (angrykoala) [inline README.md]: Is this change on purpose? :thinking:

File: README.md [hunk 2]
```diff
@@ -2,14 +2,33 @@
 
 💡 Welcome to the Monorepo for [Neo4j](https://neo4j.com/) + [GraphQL](https://graphql.org/).
 
+___
+
+***GraphQL API for Aura**
```
Comment (angrykoala) [inline README.md]:
```suggestion
**GraphQL API for Aura**  
```

**Comment by LackOfMorals** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by LackOfMorals** [comment]
recheck

File: README.md (hunk 1 above)
Comment (LackOfMorals) [reply README.md]: I don't recognise that - it's not part of the markdown I added.   Confused.

Comment (LackOfMorals) [reply README.md]: I've removed that odd character.

**Review by angrykoala — APPROVED** [review]
